src/plugins/mavlink_ftp/mavlink_ftp_impl.cpp

```diff
@@ -0,0 +1,971 @@
+#include <functional>
+#include <iostream>
+#include <experimental/filesystem>
+
+#include "crc32.h"
+#include "mavlink_ftp_impl.h"
+#include "system.h"
+#include "global_include.h"
+
+namespace mavsdk {
+
+using namespace std::placeholders; // for `_1`
+
+namespace fs = std::experimental::filesystem;
```

Comment:
Nice, this seems to make it quite easy without boost, right?

Reply:
It is being changed in newer gcc versions to std::filesystem only. I guess I should address this in cmake.

Reply:
No that's fine with me.